src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -158,10 +158,10 @@
                 <attribute name="messenger">input</attribute>
                 <attribute name="input">Sylius\Bundle\ApiBundle\Command\Checkout\ChoosePaymentMethod</attribute>
                 <attribute name="denormalization_context">
-                    <attribute name="groups">cart:select_payment_method</attribute>
+                    <attribute name="groups">shop:cart:select_payment_method</attribute>
                 </attribute>
                 <attribute name="normalization_context">
-                    <attribute name="groups">checkout:read</attribute>
+                    <attribute name="groups">shop:checkout:read</attribute>
```

Comment:
```suggestion
                    <attribute name="groups">shop:cart:read</attribute>
```
IMO we should have different groups only for cart and order, or even only for order, I'm not sure, but the serialization group for checkout seems to be unnecessary